datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -17,43 +17,36 @@
 
 //! Execution plan for reading Parquet files
 
+use arrow::datatypes::SchemaRef;
 use fmt::Debug;
```

Comment:
this file is still pretty massive, but after this PR it is down to under 2000 lines so that is progress I think

Reply:
Great! before you split. I open this file in Clion it always lagging 😂.  Thanks for your work @alamb 😄